panels/site_sections/panel_app_management.py

```diff
@@ -231,3 +231,26 @@ def everything(self, out, session):
                 ' / '.join(app.tech_needs_labels) + (' / ' if app.other_tech_needs else '') + app.other_tech_needs,
                 app.applied.strftime('%Y-%m-%d')
             ] + panelists)
+
+    def staff_schedule(self, session, attendee_id):
+        attendee = session.attendee(attendee_id)
+        events = defaultdict(lambda: defaultdict(lambda: (1, '')))
+        for pa in attendee.panel_applications:
+            if pa.event is not None:
+                for timeslot in pa.event.half_hours:
+                    rowspan = pa.event.duration if timeslot == pa.event.start_time else 0
+                    events[timeslot][pa.event.location_label] = (rowspan, pa.event.name)
+
+        schedule = []
+        locations = sorted(set(sum([list(locations) for locations in events.values()], [])))
```

Comment:
If this looks confusing at first glance, then maybe we should add a ``flatten`` function to ``utils.py``.  Python doesn't have a ``flatten`` function in its standard library, so the typical approach if you want to change ``[[5, 6], [7. 8]]`` into ``[5, 6, 7, 8]`` is to say ``sum([[5, 6,], [7, 8]], [])`` which is a bit ugly but gets the job done.